csharp/ql/src/semmle/code/csharp/exprs/Expr.qll

```diff
@@ -562,7 +562,7 @@ class Cast extends Expr {
   TypeAccess getTypeAccess() { result = this.getChild(1) }
 
   /** Gets the type that the underlying expression is being cast to. */
-  Type getTargetType() { result = this.getTypeAccess().getTarget() }
+  Type getTargetType() { result = this.getType() }
```

Comment:
This is a slightly unrelated change. Implicit casts have no `type access`, and therefore they had no target type either. I think when we have a `type access`, it will always match the expression type, so for normal casts this change has no impact. For implicit casts the change would return the type to which we convert. Let me know if my assumption here is incorrect.